Scripts/SubComponents/FilePickerLineEdit.gd

```diff
@@ -0,0 +1,57 @@
+class_name FilePickerLineEdit extends LineEdit
+
+
+signal new_file_path(file_path: String, is_valid: bool)
+
+@export var filters: Array[String]
+@export var warn_label: Label
+
+@onready var file_picker_btn: FilePickerButton = $FilePickerButton
+
+var is_valid: bool = true
+
+
+func _ready() -> void:
+	assert(warn_label != null)
+	warn_label.hide()
+
+
+func _convert_to_relative() -> void:
+	# TODO: Make MonologueControl global to access it anywhere!
```

Comment:
agree with sourcery-ai on this. not a good idea to make MonologueControl global autoload. seems like you used MonologueControl to get the current graph edit's `file_path`, which is backwards in my opinion. FilePickerLineEdit should already know what graph edit it is in because of the graph node it is supposed to represent. a better way would just be to pass the `file_path` into FilePickerLineEdit from SentenceNodePanel like this:

```
# SentenceNodePanel.gd

_ready():
    voiceline_line_edit.file_path = graph_node.get_parent().file_path
```